api/integrations/slack/authentication.py

```diff
@@ -7,21 +7,28 @@
 
 from users.models import FFAdminUser
 
+logger = logging.getLogger(__name__)
+
 
 class OauthInitAuthentication(BaseAuthentication):
     """
     Custom authentication class to use signed user_id present in query params
     """
 
     def authenticate(self, request):
+        logger.debug("OauthInitAuthentication authenticate called")
         signer = TimestampSigner()
-        with suppress(BadSignature, ObjectDoesNotExist):
+        try:
             signature = request.GET.get("signature")
```

Comment:
I would probably add a log message to describe the signature. Maybe we shouldn't print the signature itself but we should log whether it existed or not perhaps?